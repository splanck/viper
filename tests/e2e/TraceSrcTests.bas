10 PRINT "A"
20 LET B = 1
30 PRINT B
40 END
