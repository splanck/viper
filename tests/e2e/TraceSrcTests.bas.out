[SRC] <unknown>  (fn=@main blk=entry ip=#0)
[SRC] <unknown>  (fn=@main blk=entry ip=#1)
[SRC] trace_src.bas:1:4  (fn=@main blk=L10 ip=#0)  LET A = 1
[SRC] trace_src.bas:1:4  (fn=@main blk=L10 ip=#1)  LET A = 1
[SRC] trace_src.bas:2:10  (fn=@main blk=L20 ip=#0)  A
[SRC] trace_src.bas:2:4  (fn=@main blk=L20 ip=#1)  PRINT A
[SRC] trace_src.bas:2:4  (fn=@main blk=L20 ip=#2)  PRINT A
[SRC] trace_src.bas:2:4  (fn=@main blk=L20 ip=#3)  PRINT A
[SRC] trace_src.bas:2:4  (fn=@main blk=L20 ip=#4)  PRINT A
[SRC] trace_src.bas:3:4  (fn=@main blk=L30 ip=#0)  END
